winml/test/model/model_tests.cpp

```diff
@@ -224,6 +224,7 @@ static std::vector<ITestCase*> GetAllTestCases() {
   };
   allDisabledTests.insert(std::begin(x86DisabledTests), std::end(x86DisabledTests));
 #endif
+allDisabledTests.insert(ORT_TSTR("cntk_simple_seg"));
```

Comment:
I have prepared the updated cntk_simple_seg by the corrected ORT. I plan to filter it out in this PR. Then after merge I can update this model directly and propose another PR to add it back. In that case it won't cause other PRs to fail.